docs/getting_started/installation/installation.rst

```diff
@@ -17,6 +17,11 @@ after which you can simply install without specifying the channels::
 .. note::
     The order in which channels are added matters: ``conda-forge`` should be the highest priority as a result of being added last. In your ``.condarc`` file, it should be listed first.
 
+.. note::
+    Because ``packmol`` binaries are unavailable for windows from ``conda-forge`` channel to use mbuild with conda in a Windows system requires ``omnia`` channel. Use the following command to use mbuild with conda in a Windows system::
```

Comment:
```suggestion
    Because ``packmol`` binaries are unavailable for windows from ``conda-forge`` channel, to use mbuild with conda in a Windows system requires the ``omnia`` channel. Use the following command to use ``mbuild`` with conda in a Windows system::
```